starknet_py/contract.py

```diff
@@ -193,11 +195,20 @@ async def invoke(
         specified when using `Contract.prepare`
         :return: InvokeResult
         """
+        if auto_estimate and max_fee is not None:
+            raise ValueError(
+                "Both max_fee and auto_estimate cannot be provied at the same time."
```

Comment:
"max_fee and auto_estimate are exclusive."

Reply:
Done